curvefs/src/client/rpcclient/task_excutor.h

```diff
@@ -222,6 +238,27 @@ class TaskExecutorDone : public google::protobuf::Closure {
     int code_;
 };
 
+class BatchGetInodeAttrTaskExecutorDone : public TaskExecutorDone {
+ public:
+    BatchGetInodeAttrTaskExecutorDone(
+        const std::shared_ptr<TaskExecutor> excutor,
+        MetaServerClientDone *done) : TaskExecutorDone(excutor, done) {}
+    virtual ~BatchGetInodeAttrTaskExecutorDone() {}
+
+    void Run() override;
+
+    void SetInodeAttrs(const RepeatedPtrField<InodeAttr>& inodeAttrs) {
+        inodeAttrs_ = inodeAttrs;
+    }
+
+    const RepeatedPtrField<InodeAttr>& GetInodeAttrs() const {
+        return inodeAttrs_;
+    }
+
+ private:
+    RepeatedPtrField<InodeAttr> inodeAttrs_;
```

Comment:
why define them twice? the base class already defined.

Reply:
BatchGetInodeAttrTaskExecutorDone is not needed and overwrite.